### PR TITLE
Article about the size increase of binaries when using Mandrel 23.0

### PR DESCRIPTION
**Pull request by zakkak**
The article was written in a Jupyter notebook and exported to markdown using:

```
jupyter nbconvert quarkus-size-22-3-23-0.ipynb --to markdown --NbConvertApp.output_files_dir=.
```

I tried converting it to asciidoc with:

```
jupyter nbconvert quarkus-size-22-3-23-0.ipynb --to asciidoc --NbConvertApp.output_files_dir=.
```

but the output was not good. Same with `pandoc` conversion using:

```
pandoc -t asciidoc -f markdown quarkus-size-22-3-23-0.md -o _posts/2023-10-26-mandrel-23-0-image-size-increase.adoc
```

so I leave it as markdown.

### CODE REVIEW

File: _posts/2023-10-26-mandrel-23-0-image-size-increase.md [hunk 1]
```diff
@@ -0,0 +1,569 @@
+---
+layout: post
+title: 'Exploring why native executables produced with Mandrel 23.0 are bigger than with Mandrel 22.3'
+date: 2023-10-26
+tags: native graalvm mandrel metrics
+synopsis: 'A comprehensive analysis attributing binary size increase to specific changes in Mandrel's code base.'
```
Comment (maxandersen) [inline _posts/2023-10-26-mandrel-23-0-image-size-increase.md:6]:
```suggestion
synopsis: "A comprehensive analysis attributing binary size increase to specific changes in Mandrel's code base."
```

**Review by galderz — APPROVED** [review]
Looks good. @zakkak Have you got any plans to update the native ref guide? I think the section in https://quarkus.io/guides/native-reference#inspecting-native-executables could be expanded to touch on the techniques here.

**Comment by zakkak** [comment]
Uploading a screenshot since surge is not working.

image

@maxandersen can you please have a look and approve if everything looks OK?

**Comment by holly-cummins** [comment]
@zakkak, [a fix](https://github.com/quarkusio/quarkusio.github.io/pull/1826) that *should* fix previews was merged yesterday night, so if you rebase and force push, that should trigger another build and (hopefully) a working preview.

**Comment by zakkak** [comment]
The force push didn't help. Please see the screenshot above for a "preview"

**Comment by zakkak** [comment]
Yeah, we finally have a surge review https://quarkus-website-pr-1818-preview.surge.sh/blog/mandrel-23-0-image-size-increase/

**Pull request merged**